system/include/emscripten/threading.h

```diff
@@ -108,6 +116,61 @@ void *emscripten_sync_run_in_main_thread_2(int function, void *arg1, void *arg2)
 void *emscripten_sync_run_in_main_thread_3(int function, void *arg1, void *arg2, void *arg3);
 void *emscripten_sync_run_in_main_thread_7(int function, void *arg1, void *arg2, void *arg3, void *arg4, void *arg5, void *arg6, void *arg7);
 
+typedef void (*em_func_v)(void);
```

Comment:
Yeah, those can be expanded when necessary, but I couldn't find a way to make it automated that wouldn't require any kind of manual definition. (C++11 variadic templates might be able to do it, but that would be a C++ only solution, and possibly overkill)